### ISSUE TITLE
how to convert older Lamw projects to The newest version of Lamw

### ISSUE THREAD

**Issue by JanuszChmiel** [jmpessoa/lazandroidmodulewizard#104]
Please, how to fix errors which are being cause by different older version of Lamw. For demonstration, try to open this project and use CTRL+F1 command in newest Lamw package. How to solve this problem?

https://github.com/r1me/LAMWBassStreamPlayback
I had to use very old Lazarus to build this project successfully.

**Comment by jmpessoa** [comment]
Indeed,  LAMW will try handle [and update]  an "old" project ....  

I do a minor change to "smartdesigner.pas" do handle  others/exotic ".so" included in LAMW project, 
fixing the Linker error!!

Please,  update LAMW from gitub... and you will get  "LAMWBassStreamPlayback" working!!!

Thank  you!

**Comment by JanuszChmiel** [comment]
Oh you are wonderfull. Thank you very much for all new features, which 
you have implemented. And thank you very much for all issues which you 
have solved. Very very well done.
I could also use many demo examples. So thank you very much for yours 
hard work on Lazandroid module wizard project.
